lib/sam/common/pmc_common_3s.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2015 Owen Kirby <oskirby@gmail.com>
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <libopencm3/sam/pmc.h>
+
+void pmc_pllb_config(uint8_t mul, uint8_t div)
+{
+	CKGR_PLLBR = ((mul - 1) << 16) | CKGR_PLLBR_PLLBCOUNT_MASK | div;
```

Comment:
If you're handling the -1 yourself, you should _absolutely_ have that at _least_ in the doxygen...  And of course, you should have the doxygen too :)